### ISSUE TITLE
Use a catch-all try/except in the /upload route

### ISSUE THREAD

**Issue by ajgreenb** [ajgreenb/uploader#7]
It should just log the exception and `abort(406)` to prevent Mailgun from ever retrying a webhook.

Also should determine if this is a terrible idea. Is there ever a situation where I actually *want* Mailgun to retry?

**Comment by ajgreenb** [comment]
This might be the way to go:

```
try:
  perform_some_action()
except Exception as e:
  logging.exception('Failed to perform some action')
  raise e
```

and in `/upload`:

```
try:
  ...
except Exception:
  abort(406)
```

* [Catching all exceptions](https://stackoverflow.com/a/4992124/4715766)
* [Best way to log exceptions](https://stackoverflow.com/a/5191885/4715766)

**Comment by ajgreenb** [comment]
Or maybe, if custom messages don't matter:

```
perform_some_action()
```

and

```
try:
  ...
except Exception:
  logging.exception()
  abort(406)
```

**Edit:** This won't work, because `logging.exception()` requires a `msg` argument.

**Comment by ajgreenb** [comment]
Closed by d008e5b10fb0ce2ef4db9983d6d3a616e9d5b107
